scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -452,4 +512,230 @@ trait AndroidAppModule extends JavaModule {
     PathRef(Task.dest)
   }
 
+  /** The name of the virtual device to be created by  [[createAndroidVirtualDevice]] */
+  def virtualDeviceIdentifier: String = "test"
```

Comment:
Let's prefix most of the task names here with `android`. `AndroidAppModule` is expected to be mixed into `JavaModule` or `KotlinModule` or `ScalaModule`, so it would be good to prefix the tasks to avoid conflicts.

Reply:
done, let me know if I missed any